modules/backend/classes/NavigationManager.php

```diff
@@ -183,10 +202,6 @@ public function registerCallback(callable $callback)
      */
     public function registerMenuItems($owner, array $definitions)
     {
-        if (!$this->items) {
-            $this->items = [];
-        }
```

Comment:
Why was this removed?

Reply:
@LukeTowers it's not needed, I don't believe. The `loadItems` method resets the `items` and `quickActions` properties to empty arrays, and that's fired before any callbacks or events are processed.